src/components/books.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import data from './bookData';
+import Options from './options';
+
```

Comment:
- [ ] There should exist a BookList component (Which I think should be this one) which then calls the <Book> component to render your list of books, currently, you are doing both things in the same place and not using a <Book> component to handle this as required by the project, please do so to comply with the requirements.